Marlin/configuration_store.cpp

```diff
@@ -343,22 +362,15 @@ void MarlinSettings::postprocess() {
         "MBL Z array is the wrong size."
       );
       const bool leveling_is_on = TEST(mbl.status, MBL_STATUS_HAS_MESH_BIT);
-      const uint8_t mesh_num_x = GRID_MAX_POINTS_X, mesh_num_y = GRID_MAX_POINTS_Y;
       EEPROM_WRITE(leveling_is_on);
       EEPROM_WRITE(mbl.z_offset);
-      EEPROM_WRITE(mesh_num_x);
-      EEPROM_WRITE(mesh_num_y);
```

Comment:
Yeah, these changes were made because I went to a global max x & y.  Better to leave them the way they were if we're going to keep the separate max sizes.